operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -54,7 +54,8 @@ kiali_defaults:
       istio_sidecar_annotation: "sidecar.istio.io/status"
       url_service_version: ""
     jaeger:
-      service: "jaeger-query"
+      service: "tracing"
+      jaeger_service: "jaeger-query"
```

Comment:
Two things here:
1. You are missing service_namespace. Even if empty string, explicitly specify it. Virtual all of these settings should be set to their actual defaults - even if empty string.
2. Alphabetize the list. All settings are in alphabetical order. So under jaeger, you should list the children settings in this order: jaeger_service, service, service_namespace, url